src/main/java/ro/societateahermes/backendservice/entities/form/Answer.java

```diff
@@ -0,0 +1,33 @@
+package ro.societateahermes.backendservice.entities.form;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Answer {
+
+    @Id
+    @GeneratedValue
+    private long idAnswer;
+
+    @OneToOne
+    @JoinColumn(name="question_id")
+    private Question question;
+
+    @ManyToOne
+    @JoinColumn(name = "submission_id")
+    private Submission submission;
+
+    @ElementCollection
+    private List<String> textAnswer;
+
+
+
```

Comment:
Foate bine, dar am o intrebare. De ce este textul asnwerului o lista?

Reply:
E o lista deoarece intrebarea poate avea raspuns multiplu, daca e o intrebare de tip checkbox....si atunci in lista tin varianteele de raspuns bifate, pentru restul tipurilor de intrebari lista va avea doar un element.

Reply:
Ok. Foarte bine gandit atunci. Bravo 🎉